### PR TITLE
Fix deprecation warnings

### PR DESCRIPTION
**Pull request by al3xliu**

### CODE REVIEW

File: src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java [hunk 1]
```diff
@@ -20,21 +21,38 @@
 import org.checkerframework.common.value.qual.BoolVal;
 import org.checkerframework.common.value.qual.StringVal;
 import org.checkerframework.framework.type.AnnotatedTypeMirror;
-import org.checkerframework.javacutil.AnnotationUtils;
-import org.checkerframework.javacutil.BugInCF;
-import org.checkerframework.javacutil.TreeUtils;
-import org.checkerframework.javacutil.TypeAnnotationUtils;
-import org.checkerframework.javacutil.TypesUtils;
+import org.checkerframework.javacutil.*;
```
Comment (wmdietl) [inline src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java:24]: Don't do wildcard imports.

File: src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java [hunk 2]
```diff
@@ -20,21 +21,38 @@
...
 
 public class CryptoVisitor extends BaseTypeVisitor<CryptoAnnotatedTypeFactory> {
 
     final boolean STRONG_BOX_BACKED_ENABLE = checker.getLintOption("strongboxbacked", false);
 
     final ProcessingEnvironment env;
```
Comment (wmdietl) [inline src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java:30]:
```suggestion
    protected final ProcessingEnvironment env;
```

File: src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java [hunk 3]
```diff
@@ -20,21 +21,38 @@
...
-import org.checkerframework.javacutil.AnnotationUtils;
-import org.checkerframework.javacutil.BugInCF;
-import org.checkerframework.javacutil.TreeUtils;
-import org.checkerframework.javacutil.TypeAnnotationUtils;
-import org.checkerframework.javacutil.TypesUtils;
+import org.checkerframework.javacutil.*;
 
 public class CryptoVisitor extends BaseTypeVisitor<CryptoAnnotatedTypeFactory> {
 
     final boolean STRONG_BOX_BACKED_ENABLE = checker.getLintOption("strongboxbacked", false);
```
Comment (wmdietl) [inline src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java:28]:
```suggestion
    protected final boolean STRONG_BOX_BACKED_ENABLE = checker.getLintOption("strongboxbacked", false);
```

**Review by wmdietl — CHANGES_REQUESTED** [review]
Thanks! Just some small things.

File: src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java (hunk 1 above)
Comment (al3xliu) [inline src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java:24]:
```suggestion
import org.checkerframework.javacutil.AnnotationUtils;
import org.checkerframework.javacutil.BugInCF;
import org.checkerframework.javacutil.TreeUtils;
import org.checkerframework.javacutil.TypeAnnotationUtils;
import org.checkerframework.javacutil.TypesUtils;
import org.checkerframework.javacutil.AnnotationBuilder;
```

Comment (al3xliu) [reply src/main/java/org/checkerframework/checker/crypto/CryptoVisitor.java]: Thanks, resolved.

**Review by wmdietl — APPROVED** [review]
Thanks!

**Pull request merged**